CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.4.0] - 2020-10-26
+
+- Added a workaround for an NHAIS issue where the DTM segment contains a timestamp is the past. The adaptor now reads its
+ translationTimestamp from the interchange header.
+- Encapsulate outboundState recep fields in sub-document. Updated the documentation for reporting on missing RECEPs. No
+ data migration in place; to generate reports on pre-1.4.0 records follow the instructions from version 1.3.1.
+- The inboundState and outboundState.recep documents now contain a 'processedTimestamp' as a record of when the adaptor 
+ received and processed the transaction.
+- Adds a ConversationId header to outbound API requests, inbound GP system message queue, related internal queues, 
+ and database documents. Logs related to processing that transaction include the ConversationId value.
+- Replaced an embedded trust store with the capability to download a trust store (Java Keystore) at start-up when 
```

Comment:
I would add info here that the custom trust store is combined with the standard java one. Not replacing it

Reply:
👍 added a sentence to the README to clarify that the additional certificates are added to the default trust store.